scripts/index_d_ts

```diff
@@ -466,9 +467,8 @@ declare module "SendbirdUIKitGlobal" {
     onSearchClick?(): void;
     replyType?: ReplyType;
     queries?: ChannelQueries;
-    renderUserProfile?: (props: RenderUserProfileProps) => React.ReactNode | React.ReactElement;
+    renderUserProfile?: (props: RenderUserProfileProps) => React.ReactElement;
```

Comment:
This change necessary?

Reply:
ah, seems this is overlapped while I copy the interface from the Channel file, let me make it back